upi/aws/cloudformation/05_cluster_master_nodes.yaml

```diff
@@ -89,6 +89,10 @@ Parameters:
 Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
+    - Label:
+        default: "Cluster Information"
+      Parameters:
+      - InfrastructureName
```

Comment:
Why pull this in?  You don't seem to be adding consumers.

Reply:
This just adds human labels to some variables. Still missing on this rebase.